### PR TITLE
remove references to python 2.7

### PR DESCRIPTION
**Pull request by deronnax**

### CODE REVIEW

**Comment by deronnax** [comment]
@pitbulk can I have workflow approval 🥹 ?

File: setup.py [hunk 1]
```diff
@@ -23,6 +22,7 @@
         'Programming Language :: Python :: 3.11',
         'Programming Language :: Python :: 3.12',
     ],
+    python_requires='>=3.5',
```
Comment (sergei-maertens) [inline setup.py:25]: is there any reason why Python versions that are no longer receiving (security) updates should still be supported?

Python 3.8 will be EOL in October 2024, so I would aim to support only Python 3.8+.

Comment (deronnax) [reply setup.py:25]: I am just doing a slight cleanup, aligning the metadata with what the project claims. I could totally remove deprecated python version support in a different dedicated PR.

@sergei-maertens do you have the merge permission?

Comment (sergei-maertens) [reply setup.py:25]: I do not, I'm mostly tracking upstream changes to maintain our own fork

Comment (deronnax) [reply setup.py:25]: I start to think the project is abandonned.

**Pull request merged**